integrations/clickup/src/handler.ts

```diff
@@ -0,0 +1,38 @@
+import { ClickUpClient } from './client'
+import * as bp from '.botpress'
+
+export const handler: bp.IntegrationProps['handler'] = async ({ req, ctx, client, logger }) => {
+  if (!req.body) {
+    return
+  }
+
+  const clickup = new ClickUpClient(ctx.configuration.apiKey, ctx.configuration.teamId)
+
+  const body = JSON.parse(req.body)
```

Comment:
**nitpick:** `JSON.parse` could fail if the request body was not valid JSON. In this case, you may want to return a better error. I personally wrap such usage of `JSON.parse` with a try catch